pkg/tuple/tuple.go

```diff
@@ -325,3 +325,64 @@ func IsTypedWildcard(s string) bool {
 func TypedPublicWildcard(objectType string) string {
 	return BuildObject(objectType, Wildcard)
 }
+
+// MustParseTupleString attempts to parse a relationship tuple specified
+// in string notation and return the protobuf TupleKey for it. If parsing
+// of the string fails this  function will panic. It is meant for testing
+// purposes.
+//
+// Given string 'document:1#viewer@user:jon', return the protobuf TupleKey
+// for it.
+func MustParseTupleString(s string) *openfgav1.TupleKey {
```

Comment:
Introduced the helpers here to help us start using the relationship tuple notation `object#relation@user` in our codebase. I wanted to do so in this PR, so I just introduced these in this.